helm/alfresco-identity-service/values.yaml

```diff
@@ -4,6 +4,8 @@ ingress:
 
 ingressHostName: "your ingress dns name"
 
+redirectUrl: "http://localhost:*"
```

Comment:
This should be a more specific value name, something like `alfrescoClientRedirectUris`.

We'll also need to update the README on how to use it.

Reply:
👍